sass/tji/_filters.scss

```diff
@@ -0,0 +1,49 @@
+/*--------------------------------------------------------------
+>>> EXPLORE THE DATA TABLE OF CONTENTS:
+----------------------------------------------------------------
+# GENERAL
+# FILTERS
+
+/*--------------------------------------------------------------
+## FILTERS
+--------------------------------------------------------------*/
+
+#js-TJIfilters {
+  fieldset {
+    margin-bottom: 1em;
+  }
+  legend {
+    text-transform: uppercase;
+
+    .fa-caret-down { margin-left: 0.5em; }
+  }
+  legend:after {
+    font-family: 'Font Awesome 5 Free';
+    font-weight: 900;
+    content: '\f0d7'; /* Unicode character for "down" caret (+) */
+    padding-left: 0.5em;
+  }
+  legend:hover {
+    color: #F1AB32;
+    cursor: pointer;
+  }
+  .js-filter-set {
+    a { cursor: pointer; }
```

Comment:
Instead of nesting and adding styles to tags, for better CSS scoping and readability, could we lean on using [BEM syntax](http://getbem.com/introduction/) for styles. https://css-tricks.com/bem-101/
@LayaTaal this might be cool for you to give a look too.